### PR TITLE
Adds support for arbitrary signing script

### PR DESCRIPTION
**Pull request by umbernhard**
This lets us using TPMs or other signing oracles to sign data.

### CODE REVIEW

File: src/utils/options.ts [hunk 1]
```diff
@@ -75,6 +77,14 @@ async function parseOptionsWithoutTryCatch(
         return { error: new Error(`expected value for option: ${arg}`) };
       }
       signifySecretKey = value;
```
Comment (umbernhard) [inline src/utils/options.ts:79]: @benadida I left in the signify functionality here, but since I replaced it elsewhere should I take it out?

Comment (umbernhard) [reply src/utils/options.ts]: done

File: src/ipc/sign.ts [hunk 2]
```diff
@@ -30,8 +30,9 @@ function sign(
 
   try {
     const { stdout, stderr } = exec(
-      'signify-openbsd',
-      ['-S', '-s', signifySecretKey, '-m', '-', '-x', '-'],
+      // TODO this feels really dangerous, is there a better way?
```
Comment (benadida) [inline src/ipc/sign.ts:33]: Are you worried about... input sanitization? Something else?

**Review by benadida — APPROVED** [review]

**Comment by benadida** [comment]
tests are passing on local, CI is broken and we'll follow up:
https://github.com/votingworks/kiosk-browser/issues/125

**Pull request merged**